tests/testthat/test-toolGetMapping.R

```diff
@@ -38,4 +38,12 @@ test_that("toolGetMapping works", {
   writeLines("abc", paste0(getConfig("mappingfolder"), "/test.xyz"))
   expect_error(toolGetMapping("test.rda", where = "mappingfolder"), "did not contain a object")
   expect_error(toolGetMapping("test.xyz"), "Unsupported filetype")
+
+  readTest <- function() {
+    toolGetMapping("test.csv", type = "regional")
```

Comment:
If you use one of the already existing mappings you can omit the somewhat large new test.csv file.
```suggestion
    toolGetMapping("regionmappingH12.csv", type = "regional")
```